packages/mobile/src/send/SendAmount.tsx

```diff
@@ -210,14 +217,32 @@ function SendAmount(props: Props) {
     CURRENCY_ENUM.DOLLAR
   )
 
+  const onReviewButtonPressed = async () => {
+    setReviewButtonPressed(true)
+    // Wait until verification status is known to proceed
+    while (recipientVerificationStatus === RecipientVerificationStatus.UNKNOWN) {
+      await sleep(250)
+    }
```

Comment:
I don't understand how this can work.
`recipientVerificationStatus` is a new var declared and set on each render, once you press the button, it's value will never change.
What am I missing?